docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -75,43 +75,43 @@ where the Setup option is specifying the DIRAC Setup name within which the clien
 The ConfigurationServer option is used to define the URL of the Configuration Service
 that the client will contact to discover all the DIRAC services.
 
-The exact values of the command options are specific for a given user community, ask the
-group administrators for the details. Typically, a single community specific installation scripts are
-provided which are including all the necessary specifications.
+The exact values for the command options are specific to a given user community, ask the
+group administrators for its details. Typically, single community specific installation scripts are
+provided, which includes all the necessary specifications.
 
 In any case, save a "install.cfg" file with the content desired.
 
-At this point, in order to run the *dirac-configure* command, you need either a user proxy, or a host certificate.
-They are needed because dirac-configure will take care of updating the local configuration,
-but also because it will download the CAs used for connecting to DIRAC services
+At this point, in order to issue the *dirac-configure* command, you need either a user proxy, or a host certificate.
+These are needed because dirac-configure will take care of updating the local configuration as well as 
```

Comment:
The proxy is really only needed, because it connects to a DIRAC server, to update the local configuration that would not be necessary, this the "but" in the text.